### ISSUE TITLE
Method name refactor background color issue.

### ISSUE THREAD

**Issue by egavard** [ChrisRM/material-theme-jetbrains#540]
### Summary
First of all, thank you for your high quality work. It's a pleasure to work with your theme.
My problem is that I'm using material dark theme and I have some trouble while using rename feature.
The color scheme used is a bit unreadable.
The text have a yellow background while the caret is orange. The only way to know where the caret is  is to select some text. 

Thanks
...

### Expected behavior

Before moving caret, the method name is bordered with red, with a yellow caret. That seems to be good.

### Actual behavior
After MAJ+F6, before moving caret
![capture du 2017-12-19 10-12-08](https://user-images.githubusercontent.com/7823622/34149384-29463042-e4a5-11e7-96a2-14a0e8218d25.png)
After moving caret
![capture du 2017-12-19 10-12-21](https://user-images.githubusercontent.com/7823622/34149401-399c5fde-e4a5-11e7-97ab-f0f90aad28a7.png)
On text selection
![capture du 2017-12-19 10-12-36](https://user-images.githubusercontent.com/7823622/34149407-3f50d20c-e4a5-11e7-8ca5-4afda68fca25.png)

### Environment

```
IntelliJ IDEA 2017.3.1 (Ultimate Edition)
Build #IU-173.3942.27, built on December 11, 2017
Subscription is active until December 20, 2017
JRE: 1.8.0_151-8u151-b12-1~deb9u1-b12 amd64
JVM: OpenJDK 64-Bit Server VM by Oracle Corporation
Linux 4.9.0-4-amd64
```
```
<application><component name="MaterialThemeConfig"><option name="version" value="1.4.3"/><option name="highlightColor" value="80cbc4"/><option name="highlightColorEnabled" value="true"/><option name="accentColor" value="80cbc4"/></component></application>
```

**Comment by mallowigi** [comment]
Because when initiating the Refactor feature, the "identifier in caret" is highlighted in yellow. The black is actually representing the selected text, which means that typing when text is selected will replace the text with your input.

I dont have a fix for this since the caret AND the identifier under caret must be yellow. Maybe I'll think of changing that but I need to think it thoroughly as it breaks the usual styles.

**Comment by mallowigi** [comment]
Fixed it in latest version (1.4.4)
